packages/labs/virtualizer/package.json

```diff
@@ -9,34 +9,52 @@
   "type": "module",
   "exports": {
     ".": {
-      "default": "./lit-virtualizer.js"
+      "default": "./lit-virtualizer.js",
+      "types": "./lit-virtualizer.d.ts"
```

Comment:
"types" should come first in "exports" per https://www.typescriptlang.org/docs/handbook/esm-node.html#packagejson-exports-imports-and-self-referencing:~:text=The%20%22types%22%20condition%20should%20always%20come%20first%20in%20%22exports%22.

Reply:
Oh my!  Didn't realize ordering mattered in dictionary style objects.   Corrected!  Thanks for pointing it out-- I would have never suspected such!